packages/transmute-cli/components/ansible/roles/k8s-install-microservice-with-local-helm-chart/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
In a future re-factor of the existing Ansible scripts pertaining to `transmute k8s microservice install`, the role "k8s-install-microservice-with-local-helm-chart" should be used, and an additional role "k8s-install-microservice-with-helm-repo" should also be created.